packages/shared/src/lib/urls.ts

```
@@ -0,0 +1,126 @@
+];
+
+export class Urls {
+  static forRoot(): string {
+    return '/';
+  }
+
+  static forView(viewType: ViewType): string {
+    switch (viewType) {
+      case ViewType.Untriaged:
+        return this.forRoot();
+      case ViewType.Saved:
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)